test-suite/paralleltestrunner.hpp

```diff
@@ -210,7 +210,7 @@ int main( int argc, char* argv[] )
 
             std::ifstream in(profileFileName);
             if (in.good()) {
-                for (std::string line; std::getline(in, line) != 0;) {
+                for (std::string line; std::getline(in, line);) {
```

Comment:
You can prevent clang-tidy from trying to change this by adding
```suggestion
                // NOLINTNEXTLINE(readability-implicit-bool-conversion)
                for (std::string line; std::getline(in, line);) {
```
(I think that's the rule that gets triggered; if not, replace it with the one that does.)